### DISCUSSION TITLE
Stickers (the physical kind) or flyers

### DISCUSSION THREAD

**Discussion by black-puppydog** [streetcomplete/StreetComplete#4932] [Ideas]
Hey all! 
I'm moving to a new town at the moment (Toulon, southern France) and OSM still has a long way to go wrt "details". The map is *littered* with SC quests and frankly I don't have the time to do them all. (I'll try my best, but it's a big-ish place...)

I've been wondering whether there are flyers, stickers etc with streetcomplete branding. I'd try to attach them in places where they don't disturb of course, but there are plenty of places here that wouldn't suffer at all (visually) from this. Same for flyers. I don't think many people here are into OSM at all, and SC seems like the lowest possible barrier to entry. 

I have seen the [OSM materials](https://wiki.openstreetmap.org/wiki/Flyers_and_posters) and there are some adaptations to french for these as well. But they're more aimed at "real" mapping and don't have the aspect of "this is relevant to my immediate surroundings" so much, nor the "this is as simple as it gets."

Just wondering if there has been any promotional material produced specifically for SC, or whether people would be open to the idea...

**Comment by matkoniecz** [comment]
https://github.com/streetcomplete/StreetComplete/discussions/4500 has something

https://streetcomplete.app/ exists

> whether people would be open to the idea...

If you create something I would encourage you to publish materials under open license and to share how it went!

> stickers

I guess that depends on local culture (in my area it would be treated as vandalism to plaster stickers on stuff in public area, unless it is your own item)

**Reply by black-puppydog** [reply]
yeah, stickers are always a difficult subject. for starters though I'd want one for my laptop since I work in cafes quite often :)

**Comment by mnalis** [comment]
> Hey all! I'm moving to a new town at the moment (Toulon, southern France) and OSM still has a long way to go wrt "details". The map is _littered_ with SC quests and frankly I don't have the time to do them all. (I'll try my best, but it's a big-ish place...)

related to _"littered with SC quests"_: 

In `Settings` under `Manage presets`, you can create new preset (e.g. named _"only important quests"_) in which you can (via `Quest selection and display order` menu item) select only few quests that you deem most important  (for me that would be mostly cycling-related infrastructure).

Than you can easily switch later between fully-fledged `Default` preset, and this new `Only important quests` preset, depending of how full the area is of quests (and how much free time you have)

**Reply by black-puppydog** [reply]
Oh, don't get me wrong, I am comfortable with filtering these and yeah I'm focusing on bike, bus, pedestrian, and accessibility issues in my main preset (although not in that order). :P

**Comment by Cwpute** [comment]
Did you work on anything since then @black-puppydog ? i would be interested too.

**Reply by black-puppydog** [reply]
TL;DR: nope

Longer: I started working on *a* flyer. But honestly I have multiple projects I'd like to support in this way (some FLOSS, some other associations) and I started with one that's much more local and immediate. I intend to make more of these.

I *also* just asked on Mastodon whether anyone knows graphic designers/artists who'd be up for (paid) commissioned work and would "get" the spirit of the project, but didn't get much of a response. I dabbled in just finding someone on upwork to do it, but didn't get very far...

Basically, I gave the "christmas gift to myself" of an allowance of a bit of money to make this happen, but I have _never_ had to deal with this kind of question and don't even know where to start.

**Comment by Cwpute** [comment]
Thanks for your honest and quick answer.
If i ever do some of that myself i'll keep you updated.

Le 2 février 2024 14:15:23 GMT+01:00, Daan Wynen ***@***.***> a écrit :
